MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -262,6 +278,35 @@ public DepthStencilState DepthStencilState
             }
         }
 
+        internal void ApplyState(bool applyShaders)

```

Comment:
I don't remember where the `applyShaders == false` case is used anymore.  Could it be it isn't needed?